pandas/core/window/rolling.py

```diff
@@ -407,6 +415,26 @@ def hfunc(bvalues: ArrayLike) -> ArrayLike:
         self._insert_on_column(out, obj)
         return out
 
+    def _apply_tablewise(
+        self, homogeneous_func: Callable[..., ArrayLike], name: Optional[str] = None
+    ) -> FrameOrSeriesUnion:
+        if self._selected_obj.ndim == 1:
+            raise ValueError("method='table' not applicable for Series objects.")
```

Comment:
test for this?

Reply:
Yup, here: https://github.com/pandas-dev/pandas/pull/38417/files#diff-4fea2b6ab86ef8bb3c962ab4ee5667c2ea4df13bfab32ffcf03ec130451699dcR135